src/components/input/input.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../styles.scss';
+import { Label } from '../label/label';
+
+export const Input = ({
+  disabled,
+  label,
+  type,
+  name,
+  size,
+  color,
+  rounded,
+  usePlaceholder,
+  loading,
+  defaultValue,
+  onChange
+}) => {
+  const inputClassNames = ['input'];
+  const controlClassNames = ['control'];
+  if (size) inputClassNames.push(`is-${size}`);
+  if (color) inputClassNames.push(`is-${color}`);
+  if (rounded) inputClassNames.push('is-rounded');
+  if (loading) controlClassNames.push('is-loading');
+  const inputClasses = inputClassNames.join(' ');
+
+  return (
+    <div className="field">
+      {!usePlaceholder && <Label text={label} />}
+      <div className={controlClassNames.join(' ')}>
+        {!usePlaceholder ? (
+          <input
+            className={inputClasses}
+            type={type}
+            name={name}
+            disabled={disabled}
+            defaultValue={defaultValue}
+            onChange={onChange}
+          />
+        ) : (
+          <input
+            className={inputClasses}
+            type={type}
+            name={name}
+            disabled={disabled}
+            defaultValue={defaultValue}
+            placeholder={label}
```

Comment:
```suggestion
            placeholder={usePlaceholder ? label : ""}
```
Wouldn't this work instead of putting the condition around the whole input ?